src/arche/rules/result.py

```diff
@@ -138,3 +156,54 @@ def show(self, short: bool = False, keys_limit: int = 10):
 
         Report.write_summary(self)
         Report.write_rule_details(self, short=short, keys_limit=keys_limit)
+        for f in self.figures:
+            pio.show(f)
+
+    @staticmethod
+    def create_figures(stats: List[Stat]) -> List[go.FigureWidget]:
+        figures = []
+        for stat in stats:
+            if isinstance(stat, pd.Series):
+                data = [go.Bar(x=stat.values, y=stat.index.values, orientation="h")]
+            else:
+                data = [
+                    go.Bar(
+                        x=stat[c].values, y=stat.index.values, orientation="h", name=c
+                    )
+                    for c in stat.columns
+                ]
+
+            layout = go.Layout(
```

Comment:
Do you think to change the layout in future or add an option to be overriden?

Reply:
No, I don't plan to. But I will make changes once there's a need, for now I think it serves the goal perfectly.